pkg/serverless/remoteconfig/remoteconfig.go

```diff
@@ -43,6 +43,9 @@ func StartRCService(functionARN string) *remoteconfig.CoreAgentService {
 		if config.IsSet("remote_configuration.refresh_interval") {
 			options = append(options, remoteconfig.WithRefreshInterval(config.GetDuration("remote_configuration.refresh_interval"), "remote_configuration.refresh_interval"))
 		}
+		if config.IsSet("remote_configuration.org_status_refresh_interval") {
```

Comment:
```suggestion
		if config.IsConfigured("remote_configuration.org_status_refresh_interval") {
```

`config.IsSet` is an old method from viper that we're trying to move away from. It returns `true` if any possible source of configuration has a definition (so the yaml file, or an env var, or a default value from `BindEnvAndSetDefault`, etc). Using `config.IsConfigured` does what you probably want instead: returns `true` if any non-default value is assigned to the setting.